src/Storage/Commands.Storage/Azure.Storage.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '4.5.0'
+ModuleVersion = '4.5.1'
```

Comment:
Please do not increase the module version numbers. We handle version number incrementation when we create releases.